tests/scripts/thread-cert/thread_cert.py

```diff
@@ -465,6 +473,27 @@ def _collect_test_info_after_setup(self):
                 ethaddr = node.get_ether_mac()
                 test_info['ethaddrs'][i] = EthAddr(ethaddr).format_octets()
 
+    def _construct_backbone_network_name(self, backbone_network_id) -> str:
+        """
+        Construct the name of the backbone network based on the given backbone network id from TOPOLOGY. The format
+        is `backbone{PORT_OFFSET}.{backbone_network_id}`.
+
+        Args:
+            backbone_network_id: The backbone network id or None.
+
+        Returns:
+            backbone_name (str): The name of the backbone network.
+
+        Raises:
+            AssertionError: If the constructed backbone network name is not in the list of self._backbone_network_names.
+        """
+        id = backbone_network_id if backbone_network_id is not None else config.BACKBONE_DOCKER_NETWORK_DEFAULT_ID
```

Comment:
```suggestion
        id = backbone_network_id or config.BACKBONE_DOCKER_NETWORK_DEFAULT_ID
```

Reply:
they are not the same. In the "or" operator line, if `backbone_network_id` is 0, it's considered as a "Falsy" value and will return the last value here, which is not expected.

Reply:
Sounds reasonable. Thanks!

Reply:
Wait. What about having a default parameter? i.e. `backbone_network_id=config.BACKBONE_DOCKER_NETWORK_DEFAULT_ID`.

Not a blocking comment. Feel free to improve it in a future PR.